static/src/stylesheets/module/commercial/_commercial--multi.scss

```diff
@@ -109,62 +113,62 @@ Multi component (AKA Frankenponent)
     }
 }
 
-.lineitem--multi {
-    width: auto;
-    float: none;
-
-    &:nth-child(n+2) {
-        display: none;
-    }
-
-    @include mq(mobileLandscape) {
-        float: left;
-        width: 50%;
-
-        &:nth-child(n+2) {
-            display: block;
-        }
-
-        &:nth-child(n+3) {
-            display: none;
-        }
-    }
-
-    @include mq(tablet) {
-        width: percentage(1/3);
-        &:nth-child(n+3) {
-            display: block;
-        }
-
-        &:nth-child(n+4) {
-            display: none;
-        }
-    }
-
-    @include mq(desktop) {
-        &:nth-child(n+4) {
-            display: block;
+.fc-container--commercial-multi {
+    .commercial--multi {
+        .lineitem--multi {
```

Comment:
I don't like this heavy nesting but I suppose it's required for this to work in the specific situation?